<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewOrdineDipendente</class>
 <widget class="QDialog" name="ViewOrdineDipendente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ordine</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>571</width>
     <height>371</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>186</green>
         <blue>115</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>186</green>
         <blue>115</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Harlow Solid Italic</family>
     <pointsize>16</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="Carrello">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>186</green>
         <blue>115</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>186</green>
         <blue>115</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Harlow Solid Italic</family>
     <pointsize>26</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="text">
    <string>Prodotti dell'ordine</string>
   </property>
  </widget>
  <widget class="QLabel" name="Carrello_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>186</green>
         <blue>115</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>186</green>
         <blue>115</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Harlow Solid Italic</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="text">
    <string>Totale:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Carrello_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>186</green>
         <blue>115</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>206</red>
         <green>186</green>
         <blue>115</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Harlow Solid Italic</family>
     <pointsize>20</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="text">
    <string>Cod</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
